simple_imports/README.md

```diff
@@ -0,0 +1,41 @@
+# Simple Imports
+
+This folder contains configs for imports using [Simple Stats
+Importer](https://github.com/datacommonsorg/import/edit/master/simple/README.md).
+
+## Import config
+
+Each import has a sub-folder with a config file `<import-name>/simple_import.json`.
+The parameters for the simple import config file are described in detail 
+[here](https://github.com/datacommonsorg/import/blob/master/simple/stats/config.md).
+
+The config has the following parameters for automated updates:
+```
+{
+  "dataDownloadUrl": ["<GCS path containing input files>"],
+  "inputFiles": {
+    "<file1.csv>": {
+      "importType": "observations",
+      "entityType": "Country",
+    },
+  },
+  "provenanceUrl": "<url describing the import source data>",
+  "provenanceDescription": "<description including citation for the import source>",
+  "cronSchedule": "<Minute> <Hour> <Day-of-month> <Month> <Day-of-week>"
+}
+```
+
+Imports with a `cronSchedule` are processed periodically as a cloud run job by
+the [import automation](https://github.com/datacommonsorg/data/tree/master/import-automation).
+
+
+## Adding a new import
+
+To add a new data source:
+1. Copy the input files to a GCS bucket that is
+read accessible to the service account `965988403328-compute@developer.gserviceaccount.com`.
```

Comment:
Can you add the specific permissions ("Storage Legacy Bucket Owner", "Storage Legacy Object Owner", any others) needed? Also, screenshots if you have them handy (or a TODO to add them later).

Reply:
The BucketOwner and ObjectOwner are only needed for the output folder. Can the input be just read access `Storage Legacy Bucket Reader`?
https://screenshot.googleplex.com/6GmNG7fQRRE8qiN

Reply:
Probably. Consider trying with a sample input directory and documenting the appropriate permissions here.

Reply:
Tried with a GCS bucket: both Storage Legacy Bucket and Object Reader permissions are sufficient. No need for Admin access.